runtime/src/accounts_db.rs

```diff
@@ -764,22 +808,40 @@ impl AccountsDB {
         }
 
         self.handle_reclaims(&reclaims);
+        reclaims_time.stop();
+        debug!(
+            "clean_accounts: {} {} {} {}",
+            accounts_scan, store_counts_time, purge_filter, reclaims_time
+        );
     }
 
     fn handle_reclaims(&self, reclaims: &[(Slot, AccountInfo)]) {
         let mut dead_accounts = Measure::start("reclaims::remove_dead_accounts");
-        let mut dead_slots = self.remove_dead_accounts(reclaims);
+        let dead_slots = self.remove_dead_accounts(reclaims);
         dead_accounts.stop();
+        self.dead_slots.write().unwrap().extend(dead_slots);
```

Comment:
Maybe, for the cases of extreme, we should just block here when there is already pending huge number of `self_slots`, say `.len()` is like 1000 just to avoid resource exhaustion attacks. Other wise, we could grow any unbounded number of `AppendVec` (= `mmap`)s.